modules/cudaarithm/include/opencv2/cudaarithm.hpp

```diff
@@ -685,15 +685,33 @@ CV_EXPORTS_W void reduce(InputArray mtx, OutputArray vec, int dim, int reduceOp,
 
 /** @brief Computes a mean value and a standard deviation of matrix elements.
 
-@param mtx Source matrix. CV_8UC1 matrices are supported for now.
-@param mean Mean value.
-@param stddev Standard deviation value.
+@param src Source matrix. CV_8UC1 and CV_32FC1 matrices are supported for now.
+@param mask Operation mask.
+@param dst Desitnation buffer for mean and stddev.
+@param stream Cuda stream for execution.
 
 @sa meanStdDev
  */
-CV_EXPORTS_W void meanStdDev(InputArray mtx, Scalar& mean, Scalar& stddev);
-/** @overload */
+CV_EXPORTS_W void meanStdDev(InputArray src, InputArray mask, OutputArray dst, Stream& stream = Stream::Null());
```

Comment:
> InputArray src, InputArray mask, OutputArray dst

By current design, `mask` as optional input should go after required output parameter:

```
InputArray src, OutputArray dst, InputArray mask = noArrray(), Stream& stream = Stream::Null()
```